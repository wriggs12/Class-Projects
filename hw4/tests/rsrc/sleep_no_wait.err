[00000.000000] STARTUP
[00000.000031] PROMPT
[00000.100241] INPUT run echo hello world
[00000.100445] CHANGE 2187657: none -> running
[00000.101185] SIGNAL 17
[00000.101220] CHANGE 2187657: running -> stopped
[00000.101265] PROMPT
[00000.201364] INPUT run sleep 1
[00000.201575] CHANGE 2187658: none -> running
[00000.202266] SIGNAL 17
[00000.202298] CHANGE 2187658: running -> stopped
[00000.202334] PROMPT
[00000.302459] INPUT cont 1
[00000.302496] CHANGE 2187658: stopped -> running
[00000.302517] PROMPT
[00000.402659] INPUT cont 0
[00000.402700] CHANGE 2187657: stopped -> running
[00000.402729] PROMPT
hello world
[00000.404013] SIGNAL 17
[00000.404046] CHANGE 2187657: running -> dead
[00000.503011] INPUT quit
[00000.503033] CHANGE 2187658: running -> killed
[00000.503384] SIGNAL 17
[00000.503422] CHANGE 2187658: killed -> dead
[00000.503452] SHUTDOWN
