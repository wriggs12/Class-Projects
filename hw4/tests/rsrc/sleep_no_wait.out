0	2187657	T	running		echo hello world
0	2187657	T	stopped		echo hello world
1	2187658	T	running		sleep 1
1	2187658	T	stopped		sleep 1
1	2187658	T	running		sleep 1
0	2187657	T	running		echo hello world
0	2187657	T	dead	0x0	echo hello world
1	2187658	T	killed		sleep 1
1	2187658	T	dead	0x9	sleep 1
